shakedown/dcos/docker.py

```diff
@@ -9,6 +9,53 @@
 from dcos import marathon
 
 
+def docker_version(host=None, component='{{.Server.Version}}'):
+    """ Return the version of Docker [Server]
+
+        :param host: host or IP of the machine Docker is running on
+        :type host: str
+	:param component: Docker component
+	:type component: str
+        :return: Docker version
+        :rtype: str
+    """
+
+    # sudo is required for non-coreOS installs
+    command = 'sudo docker version -f {}'.format(component)
```

Comment:
Do we care about security here? It looks like `run_command_on_*` ultimately calls Paramiko's `exec_command(command)` which looks like a straight pass-through. So I could do something like this as root:

```
docker_version('{{.Client.Version}} && rm -rf /')
```

Maybe hard-code the `{{.XXXX.Version}}` strings in the command and have a client/server flag instead? Get all possible user input out of the equation?